index.bs

```diff
@@ -645,7 +645,20 @@ requirements.</p>
  specification. [[!WEBIDL]]</p></dd>
 
  <dt>User agents with no scripting support</dt>
- <dd><p>All processing requirements in this specification apply, except those in [[#api]].</p></dd>
+ <dd><p>All processing requirements in this specification apply, except those in
+ [[#dom-construction-rules]] and [[#api]].</p></dd>
+
+ <dt><dfn>User agents that do not support CSS</dfn></dt>
+ <dd><p>All processing requirements in this specification apply, except parts of [[#parsing]] that
+ relate to stylesheets and CSS, and all of [[#rendering]] and [[#css-extensions]]. The user agent
+ must instead only render the text inside <a>WebVTT cue text</a> in an appropriate manner. Any other
+ styling and positioning instructions are optional.</p> </dd>
+
```

Comment:
Unless I misunderstood something, we seem to go in circles.

When it says for "User agents that do not support CSS" 

> The user agent must instead only render the text inside <a>WebVTT cue text</a> in an appropriate manner. Any other styling and positioning instructions are optional.

...it seems again also exlcude the required support for color. It even goes further and makes positioning  (which is not defined by CSS extensions but by Cue settings) optional.